src/scss/styles.scss

```
@@ -0,0 +1,106 @@
+// Custom scss
+@import "../../node_modules/bootstrap/scss/functions";
+@import "./partials/variables";
+@import "./partials/mixins";
+@import "./partials/card";
+@import "./partials/btn";
+@import "./partials/nav";
+
+// Original Bootstrap scss files
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+// Custom Font-Family
+@import url("https://use.typekit.net/obt0qwc.css");
+
+// Roots
+
+:root {
+  --bs-body-font-family: open-sans, sans-serif;
+  --bs-border-width: none;
+}
+
+// Custom img styling
+.img-logo {
+  @include img-media(60%);
+}
+
+.form-control,
+.nav-link,
+.btn:hover {
+  color: $dark;
+}
+
+.form-control,
+.btn-login:hover {
+  background-color: $secondary;
+}
+
+.btn-custom:hover {
+  background-color: $light;
+}
+
+// Customized form from "../../node_modules/bootstrap/scss/forms/form-control.scss"
+.form-control {
+  width: 17rem;
```

```suggestion
  width: 100%;
  max-width: 17rem;
```

This will give better responsive behaviour. Setting a maximum bound will allow for the element to shrink when needed.

---

I'll fix this at once!

---

This has now been updated